Resources/translations/SonataUserBundle.lt.xliff

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0"?>
-<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file source-language="lt" datatype="plaintext" original="" >
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I did a manual check on an internal project with admin-bundle, this change nothing. :+1: